Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -163,6 +163,15 @@ void OnSceneLoaded(Scene scene, LoadSceneMode mode)
                     Debug.Log("Rebuild Client spawnableObjects after additive scene load: " + scene.name);
                 }
             }
+
+            // finished our async scene load that was in progress?
+            if (singleton != null && loadingSceneAsync != null)
```

Comment:
do we need "loadingSceneAsync"  anymore?

Reply:
to not break it, we need to detect if we finished an async scene load that we called serverchangescene for - so we can call FinishLoadScene

Reply:
Perhaps a better way is to use coroutines.   We start loading the other scene async,   we yield and after it comes back,  we do what we have to do.   Less state,  and will work even if we are loading multiple scenes at the same time.